aldryn_people/admin.py

```diff
@@ -17,24 +17,31 @@ class PersonAdmin(AllTranslationsMixin, TranslatableAdmin):
     list_display = [
         '__str__', 'email', 'vcard_enabled', ]
     list_filter = ['groups', 'vcard_enabled']
-    search_fields = ('name', 'email', 'translations__function')
-    prepopulated_fields = {'slug': ('name',)}
+    search_fields = ('translations__name', 'email', 'translations__function')
     raw_id_fields = ('user',)
 
+    def get_prepopulated_fields(self, request, obj=None):
```

Comment:
there's a django bug that does not load js files for the prepopulation mechanism when you use ```get_prepopulated_fields```, so you'll have to include the media manually.